src/paperless/settings.py

```diff
@@ -151,7 +151,10 @@
 # https://docs.djangoproject.com/en/1.9/howto/static-files/
 
 STATIC_ROOT = os.path.join(BASE_DIR, "..", "static")
-MEDIA_ROOT = os.path.join(BASE_DIR, "..", "media")
+MEDIA_ROOT = os.getenv(
```

Comment:
Another nitpick: pep8 rules say that all indents must be with 4  spaces.  I suggest this instead:

```python
MEDIA_ROOT = os.getenv(
    "PAPERLESS_MEDIADIR", os.path.join(BASE_DIR, "..", "media"))
```

or

```python
MEDIA_ROOT = os.getenv(
    "PAPERLESS_MEDIADIR",
    os.path.join(BASE_DIR, "..", "media")
)
```